### ISSUE TITLE
inherit disjoint relationship from CHEBI:51880

### ISSUE THREAD

**Issue by JKChang2015** [ebi-chebi/ChEBI#3268]
I reasoned the Chebi-disjoints.owl using ELK, got these issues:
![a4f37be0-5ea7-4b9d-8709-08fd21e232d4](https://cloud.githubusercontent.com/assets/16242098/22503095/f402ef5e-e867-11e6-93dc-c5b7cbb48bfd.png)

****CHEBI:51884** 'tetrahydroasterriquinone'** 
****CHEBI:51883** 'asterriquinone dimethyl ether'** and
****CHEBI:51881** 'asterriquinone'** 

doesn't inherit disjoint relationship from 
**CHEBI:51880 'asterriquinones'**

Here is the inferred result (CHEBI:51880):
![b582e5e8-78f1-486c-b67b-fbb5084fec61](https://cloud.githubusercontent.com/assets/16242098/22503308/0c5f9df8-e869-11e6-900e-3d581a7279b0.png)

**Comment by G-Owen** [comment]
The disjoint statement that says asterriquinones is disjoint with benzoquinones needs to be deleted.....

**Comment by muthuvenkat** [comment]
@JKChang2015  its done in chebi-disjoint.owl in ftp

**Comment by JKChang2015** [comment]
I ran ELK again, it seems still have some issues:
<img width="258" alt="screen shot 2017-02-16 at 14 00 02" src="https://cloud.githubusercontent.com/assets/16242098/23024056/53fb4afc-f450-11e6-9722-52ce1e0c9428.png">


CHEBI_134347
CHEBI_134301
CHEBI_134308
CHEBI_66331
CHEBI_134289

**Comment by cmungall** [comment]
I recommend using robot to validate releases. Are you already using robot to convert the obo to owl?

**Comment by muthuvenkat** [comment]
Yeah @cmungall, you are right we are using robot to convert the obo to owl.

**Comment by cmungall** [comment]
I forget does the disjoints import chebi?

if so

`robot reason -r elk -i chebi-disjoint.owl`

You may want to set up a catalog to use the local chebi rather than release

while I'm here, do you want a purl for chebi-disjoint.owl?

**Comment by cmungall** [comment]
I also recommend putting your release script in github to make it easier for 3rd party checking & execution.

**Comment by cmungall** [comment]
I just remembered we set up a CI job to check for this a long time ago, as we were intending to use the disjointness axioms in OWL: 

https://build.berkeleybop.org/view/GO/job/check-chebi-disjoints

It's been red for a while :-)

If the ontology is incoherent, it generates an unsatisfiable module, which is the minimal ontology that entails the unsatisfiable classes (it can be faster than testing the whole ontology)
